config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
   get "/menu", to: "items#index"
+  resources :cart_items, only: [:create]
+  get '/cart', to: "cart_items#index"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.